### PR TITLE
add registries create update delete query function

### PR DESCRIPTION
**Pull request by littlebeer2100**

### CODE REVIEW

File: pkg/models/nodes.go [hunk 1]
```diff
@@ -216,3 +217,9 @@ func getNodeFileSystemStatus(node *v1.Node) (string, string, string) {
 	}
 	return "", "", ""
 }
+
+func DrainNode(nodename string) {
```
Comment (calvinyv) [inline pkg/models/nodes.go]: As no logic is added in this PR for nodes model, I think you should undo this change

Comment (littlebeer2100) [reply pkg/models/nodes.go]: ok，i will remove this funcion
